samples/CloudClipboard/CloudClipboard/GarbageCollector/Properties/FutureExtensions.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using Azure.Core;
+using Azure.Core.Pipeline;
+using Azure.Storage.Blobs;
+
+namespace GarbageCollector
+{
+    // Enable future APIs today
+    public static class FutureExtensions
+    {
+        public static void AddPolicy(this BlobClientOptions options, HttpPipelinePolicy policy, HttpPipelinePosition position)
```

Comment:
Unless I'm misreading, this seems unnecessary.   The extension is just delegating to the public method on the options.    Did I overlook something?

Reply:
This isn't needed anymore.  This was a fun "point in time" workaround.   We knew we'd be changing the order of the params by GA, but we shot the videos a couple months before they aired.  We can just delete the file and everything should still compile.